### DISCUSSION TITLE
Security mode - issue at target temperature change

### DISCUSSION THREAD

**Discussion by raphael-homann** [jmcollin78/versatile_thermostat#245] [Q&A] [answered]
Bonjour, 

Tout d'abord un grand bravo pour le travail fourni ! un peu difficile à mettre en place, pour trouver le bon réglage, mais bravo !

C'est la seconde fois que je dois désactiver versatile thermostat, pour revenir à un thermostat classique, mais à grand regret. Je vous partage mon expérience, il y aura sans doute une explication...

Je règle le thermostat sur 17° pour la nuit, et à 6h30, la température cible bascule sur 20°.
toute la période a 17°, on voit bien les "impulsions" de chauffage permettant de maintenir a 17°, mais lorsque la température cible passe a 20°, au lieu de s'activer à 100%, il ralentit, s'arrête presque. J'ai donc basculé sur le thermostat classique a 8h (d'où le décalage entre le graph en jaune et le orange).
vers 7h50, le thermostat semble revenir a un fonctionnement correct.

est-ce qu'il se serait mis dans un mode de sécurité ?

![image](https://github.com/jmcollin78/versatile_thermostat/assets/7700663/2899e858-c2bf-4e94-94f3-ac6d7552237c)

**Comment by raphael-homann** [comment]
je viens de voir qu'on peut suivre l'activation du mode de sécurité. En effet il s'était bien mis en mode sécurité : 
![image](https://github.com/jmcollin78/versatile_thermostat/assets/7700663/1814edab-185a-44d6-b9f0-64cc1e0e420d)

mais je ne vois pas pourquoi

**Comment by raphael-homann** [comment]
est-ce que ce graph indiquant la "dernière date de relevée de température" montre des faiblesses ? 
je pense que, comme la température est "trop stable" (dommage, c'est ce qu'on cherche :smile:), le thermostat ne remonte pas de mise a jour, et versatile le croit hors service ?

![image](https://github.com/jmcollin78/versatile_thermostat/assets/7700663/8c6c0eb1-544b-4796-8980-19b92fdc66ed)

**Comment by jmcollin78** [comment, marked as answer]
Bonjour,

Voilà c'est exactement ce qui se passe. La température n'est plus reçue pendant longtemps, donc tant qu'il n'y a pas besoin de puissance, le mode sécurité ne s'enclenche pas (car pas de risque) mais dès que tu demandes de la puissance (passage à 20°) là il s'active. Et quand il s'active il garde un minimum de chauffe pour pas qu'il fasse 2° dans la pièce.

Tout ça se régle dans la dernière page de configuration du VTherm ("configuration avancée").

> je pense que, comme la température est "trop stable"

Oui. Sur certains thermostat, il y a possibilité de forcer un envoi de température (j'ai ça sur les TuYa). Sur les Zigbee, je n'en ai jamais eu besoin. Si tu peux le faire, force un envoi toutes les 3/4 d'h et règle le paramètre à 1 h.

C'est un peu pénible cette histoire mais je ne peux pas laisser un radiateur chauffé alors qu'on n'a plus de relevé de température. Ca pourrait devenir dangereux.

**Reply by raphael-homann** [reply]
Merci pour ta réponse. 
Ce sont des capteurs zigbee zonoff, je n'ai pas trouvé comment envoyer la température a intervalles réguliers.
ce réglage de sécurité me semble en effet nécessaire, et je ne vois donc pas de meilleurs solution, en effet...

Si je laisse en l'état, il va rester stable la nuit, puis lorsque je demande à monter, il se met en sécurité, et une fois que la température est descendue, le thermostat va remonter une nouvelle mesure, et tout va repartir. Mais en terme de réactivité, c'est pas terrible...

est-ce qu'il n'y aurait pas un moyen de basculer en mode "forcé" pendant un délai paramétrable ? en gros, lorsque la température cible monte, et qu'on n'a pas eu de nouvelles du capteur depuis trop longtemps (délai de sécurité), on active le chauffage X minutes, le temps de réactiver le thermomètre ?

en attendant mieux, J'ai monté a 10h le délai de sécurité, histoire de passer la nuit, mais en effet, je me mets en danger en cas de perte réelle de capteur...

**Comment by jmcollin78** [comment]
2 h ça peut suffire non ?. L'autre idée c'est de mettre 0,5 en cas mode sécurité, comme ça ça chauffe qd même un peu en attendant une maj de la température
